### ISSUE TITLE
Click event handler not assigned correctly on mobile browsers

### ISSUE THREAD

**Issue by djlnet** [senchalabs/jQTouch#198]
Am trying out the Kilo demo found in Jonathan Stark's Building Android Apps (http://oreilly.com/catalog/9781449383268), updated to use jQTouch revision 161 with jQuery version 1.4.2. I have it working great in a desktop browser (Google Chrome), but the click event handlers do not work correctly on the iPhone or on two Android devices (Motorola Droid X and Samsung Galaxy Tab).

You can see this a little more easily if you follow these steps:
1. Navigate to http://www.djlucas.net/android/ch07/index.html in a desktop (not a mobile) browser.
2. Click the Dates button (link).
3. Click the Today button (link). You should see two alerts, one that says, "Calling the date button click event", and another that says, "The call to set the date in the h1 header succeeded." This is correct behavior.
4. Repeat steps 1-3 from an Android phone or iPhone browser. You will notice that the alerts display on step 2, not step 3.

In other words, the event handler is correctly being assigned to the six anchor tags when the page runs in a desktop browser, but they are being assigned instead to the first anchor tag of the parent panel when the page runs on a phone.

The code that assigns the click event is as follows:

$('#dates > ul > li > a').click(function(){
    alert('Calling the date button click event.');
    var dayOffset = this.id;
    var date = new Date();
    date.setDate(date.getDate() - dayOffset);
    var currentDate = date.getMonth() + 1 + '/' +
        date.getDate() + '/' +
        date.getFullYear();
    sessionStorage.currentDate = currentDate;
    refreshEntries(currentDate);
});
